active/ScrollingView/ScrollingView.md

```diff
@@ -0,0 +1,2584 @@
+
+
+### ScrollingView, additional events
+
+| **Member** | **Description**                                                            |
+|------------|----------------------------------------------------------------------------|
+| ExtentChanged | Raised when any of the two ExtentWidth and ExtentHeight properties changed. |
+| State | Gets the current interaction state of the control | Idle |
+| ScrollAnimationStarting | Raised when a ScrollTo or ScrollBy call triggers an animation. Allows customization of that animation |
+| ScrollCompleted | Raised at the end of a ScrollTo, ScrollBy, or ScrollFrom asynchronous operation. Provides the original ScrollingScrollInfo struct |
+| ZoomAnimationStarting | Raised when a ZoomTo or ZoomBy call triggers an animation. Allows customization of that animation. |
+| ZoomCompleted | Raised at the end of a ZoomTo, ZoomBy, or ZoomFrom asynchronous operation. Provides the original ScrollingZoomInfo struct. |
+| BringingIntoView | Raised at the beginning of a bring-into-view-request participation. Allows customization of that participation.
+
+
+### ScrollingView, additional methods
+
+| **Member** | **Description**                                                            |
+|------------|----------------------------------------------------------------------------|
+| ScrollTo(double horizontalOffset, double verticalOffset) | Asynchronously scrolls to specified offsets. Allows animation, respects snap points. |
+| ScrollTo(double horizontalOffset, double verticalOffset, ScrollingScrollOptions options) |Asynchronously scrolls to specified offsets with optional animation, with optional snap points respecting.
+| ScrollBy(double horizontalOffsetDelta, double verticalOffsetDelta) | Asynchronously scrolls by the provided delta amount. Allows animation, respects snap points.
+| ScrollBy(double horizontalOffsetDelta, double verticalOffsetDelta, ScrollingScrollOptions options) | Asynchronously scrolls by the provided delta amount with optional animation, with optional snap points respecting.
+| ZoomTo(float zoomFactor,IReference<Vector2> centerPoint) | Asynchronously zooms to specified zoom factor. Allows animation.
+| ZoomTo(float zoomFactor, IReference<Vector2> centerPoint, ScrollingZoomOptions options) | Asynchronously zooms to specified offsets with optional animation.
+| ZoomBy(float zoomFactorDelta, IReference<Vector2> centerPoint) | Async
```

Comment:
There is already a `BringIntoViewRequestedEventArgs.Handled` property. How does `Cancel` differ from `Handled` here?

Reply:
ScrollingBringingIntoViewEventArgs.Cancel is set to True to skip the participation of the ScrollingView that raises the event. The request will still be processed potentially by parent ScrollViewer/ScrollingView controls. It’s to cancel the one participation that is about to be started.  
Setting BringIntoViewRequestedEventArgs.Handled to true stops the entire chain of participations.

Reply:
More precisely, ScrollingBringingIntoViewEventArgs.Cancel cancels the sender's contribution that was about to begin. It is typically set to True by app code. Potential parent scrolling controls will continue to attempt to bring the target element into their own viewports. 

BringIntoViewRequestedEventArgs.Handled is typically set to True by a scrolling control when it couldn't bring the target into its own viewport, not even partially. So parent scrolling controls don't try to bring the target into their view because it would be pointless.
When the target is successfully being moved into the participant's view, completely or partially, e.Handled remains False and the parents try to bring that participant into their view (the participant becomes the new target).

So Cancel prevents the scroll for one participant, Handled stops the scroll for all participants.

Reply:
Discussion: Scenario for Cancel is an inner ScrollingView that already has the element partially in view, and doesn't want to move beyond that. But the outer ScrollingView should still scroll everything into view.

Recommend: Update the description, explain the Cancel vs Handled behavior, give the scenario as an example.  Is this an important scenario though? Consider leaving it for a later release.